### PR TITLE
Pathplanning 🚀 🌕 

### PR DESCRIPTION
**Pull request by 5690Programmers**
- based 🥵 on connectorX branch
- removes discretize
- changes to wrist direction
- add desktop support
- update to 2024 beta (it no worky 😢 )

### CODE REVIEW

**Review by having11 — CHANGES_REQUESTED** [review]
fix conflicts and allow me to CAREFULLY review it

File: src/main/cpp/subsystems/DriveSubsystem.cpp [hunk 1]
```diff
@@ -34,6 +40,31 @@ DriveSubsystem::DriveSubsystem()
...
+        },
+        [this](frc::ChassisSpeeds speeds) -> void
+        {
+            Drive(speeds);
+        },
+        pathplanner::HolonomicPathFollowerConfig(
+            pathplanner::PIDConstants(0.5, 0.0, 0.0), // Translation PID constants
+            pathplanner::PIDConstants(0.5, 0.0, 0.0), // Rotation PID constants
+            4.5_mps,                     // Max module speed, in m/s
+            0.4_m,                       // Drive base radius in meters. Distance from robot center to furthest module.
```
Comment (having11) [inline src/main/cpp/subsystems/DriveSubsystem.cpp:64]: Will these be made into named constants?

File: src/main/cpp/RobotContainer.cpp [hunk 2]
```diff
@@ -51,9 +52,12 @@ RobotContainer::RobotContainer() {
                 m_driverController.GetLeftX(), OIConstants::kDriveDeadband)},
             -units::radians_per_second_t{frc::ApplyDeadband(
                 m_driverController.GetRightX(), OIConstants::kDriveDeadband)},
-            true, true, kLoopTime);
+            true, false, kLoopTime);
       },
       {&m_drive}));
+
+    m_chooser.SetDefaultOption("Leave Community", pathplanner::PathPlannerAuto("Leave COM").ToPtr().Unwrap().get());
```
Comment (having11) [inline src/main/cpp/RobotContainer.cpp:59]: Extract to constants?

**Review by having11 — APPROVED** [review]
Approved, but the items I pointed out will need to be addressed later

File: src/main/cpp/RobotContainer.cpp (hunk 2 above)
Comment (nobody5050) [reply src/main/cpp/RobotContainer.cpp:59]: Extract what?

Comment (having11) [reply src/main/cpp/RobotContainer.cpp:59]: The names `"Leave Community"` and `"Leave COM"`

Comment (nobody5050) [reply src/main/cpp/RobotContainer.cpp:59]: Those are arguments?

**Pull request merged**